### ISSUE TITLE
Disabled notification crash

### ISSUE THREAD

**Issue by robbely** [SamsidParty/TopNotify#39]
This isn;t an issue new to the latest version, but is one that I had before that hadn't been fixed.
I am finding if I disable notifications from a specific app in Windows Settings (system -> notifications and actions), TopNotify will crash when that app would have normally displayed a notification. 

(Also, relating to the notification sounds issue, after the crash, windows system sounds are reset to "silent.wav" when TopNotify is next run)

**Comment by SamarthMP** [comment]
Working on it...

> Also, relating to the notification sounds issue, after the crash, windows system sounds are reset to "silent.wav" when TopNotify is next run

This is intended, as TopNotify plays the sounds and silences the system from playing it

**Comment by SamarthMP** [comment]
Hmm, I'm not able to reproduce this issue, can you tell me what app specifically is causing the issue?
Also, are you on windows 10 or 11?

**Comment by robbely** [comment]
Many thanks for the reply - I'm getting the issue on Psyphon3 app using Windows 10

**Comment by SamsidParty** [comment]
Could you try installing beta 2.6.1?

**Comment by robbely** [comment]
I'm getting this error trying to install the beta (and the Install button is disabled):

`This app package’s publisher certificate could not be verified. Contact your system administrator or the app developer to obtain a new app package with verified certificates. The root certificate and all immediate certificates of the signature in the app package must be verified (0x800B010A)`

**Comment by SamsidParty** [comment]
https://www.samsidparty.com/docs/support/beta-program/sideloading-msix

**Comment by robbely** [comment]
Many thanks - are there instructions on how to remove the certificate again once a fixed release build is published?

**Comment by robbely** [comment]
I prefer to not have any security holes in my system, even if they are unlikely/low risk
I think I'll wait till the release version and try that one .

Here are the crash details in case they help:```

Problem signature:
  Problem Event Name:	MoAppCrash
  Package Full Name:	55968SamsidGameStudios.TopNotify_2.6.0.0_x64__r9j5xrxak4zje
  Application Name:	praid:TopNotify
  Application Version:	1.0.0.0
  Application Timestamp:	66960000
  Fault Module Name:	KERNELBASE.dll
  Fault Module Version:	10.0.19041.5131
  Fault Module Timestamp:	011921da
  Exception Code:	e0434352
  Exception Offset:	000000000003b699
  OS Version:	10.0.19045.2.0.0.256.4
  Locale ID:	2057
  Additional Information 1:	00c3
  Additional Information 2:	00c31b3a8bdc043497ce4fb9a1c58281
  Additional Information 3:	78c5
  Additional Information 4:	78c53e7030e46a8e17bc19d421543ef0
```

**Comment by robbely** [comment]
I haven't tried the beta yet due to the certificate (I haven't installed that yet). I'll update once I have figured out the removal process.

**Comment by SamsidParty** [comment]
New version is live on the store, please check that.

**Comment by robbely** [comment]
Many thanks, unfortunately the issue is still there.

**Comment by SamsidParty** [comment]
I see, is there an option to disable notifications in that app itself instead of windows settings?

**Comment by robbely** [comment]
Unfortunately not, most of the notifications are pop-up ads, so they want me to see them.

**Comment by SamsidParty** [comment]
Are you referring to psiphon 3 (the VPN)?
I know telling you to use another app doesn't address the problem, but I highly recommend using cloudflare warp instead.

As for the issue itself, I'll have to try and reproduce it in a virtual machine, so it'll take time to fix.

**Comment by robbely** [comment]
Many thanks for the suggestion, I found I couldn't get cloudflare warp to work how I wanted it to in the past, I might give it another go.

Re the issue, it isn't deal-breaking, if I start psyphon3 before topnotify it only crashes occasionally (if I start topnotify first it'll crash every time).

I'd be interested to know if you can reproduce the issue, but there's no need for a quick fix.

**Comment by robbely** [comment]
Did you manage to reproduce the issue with Psyphon3?

**Comment by SamsidParty** [comment]
No, I haven't actually tried because I'm currently working on a rewrite of TopNotify's underlying engine (IgniteView). I'll get to it when I can.

**Comment by robbely** [comment]
OK. I'll wait for the new engine version and report back.
